ext/pdo_mysql/tests/pdo_mysql_stmt_fetch_serialize_fetch_class.phpt

```diff
@@ -110,6 +110,10 @@ $db = MySQLPDOTest::factory();
 $db->exec('DROP TABLE IF EXISTS test_stmt_fetchserialize_fetch_class');
 ?>
 --EXPECTF--
+Deprecated: The __sleep() serialization hook has been deprecated. Implement __serialize() instead (or in addition, if support for old PHP versions is necessary) in %s on line %d
```

Comment:
Can these tests not be rewritten to use the new methods?

Reply:
No, because those tests are for the deprecated `PDO::FETCH_SERIALIZE` option which relies on the deprecated `Serialize` interface methods.